.rubocop.yml

```diff
@@ -174,28 +170,28 @@ Lint/AssignmentInCondition:
   StyleGuide: http://relaxed.ruby.style/#lintassignmentincondition
 
 Metrics/AbcSize:
-  Enabled: false
+  Max: 15
 
 Metrics/BlockNesting:
-  Enabled: false
+  Max: 3
 
 Metrics/ClassLength:
-  Enabled: false
+  Max: 100
 
 Metrics/ModuleLength:
-  Enabled: false
+  Max: 100
 
 Metrics/CyclomaticComplexity:
-  Enabled: false
+  Max: 6
 
 Metrics/LineLength:
-  Enabled: false
+  Max: 80
 
 Metrics/MethodLength:
-  Enabled: false
+  Max: 10
 
 Metrics/ParameterLists:
-  Enabled: false
+  Max: 5
 
 Metrics/PerceivedComplexity:
-  Enabled: false
+  Max: 7
```

Comment:
These are the default values. They need to be specified so that we override the ones inherited from the rubocop_todo.yml, otherwise, the largest Max value found in the codebase will be used. More details: https://github.com/rubocop-hq/rubocop/issues/3264#issuecomment-259959753